packages-ts/starknet-gauntlet/src/provider/index.ts

```diff
@@ -104,6 +115,37 @@ class Provider implements IStarknetProvider {
     return response
   }
 
+  declareContract = async (contract: CompiledContract, compiledClassHash?: string, wait = true) => {
+    const tx = await this.account.declare({
+      contract,
+      compiledClassHash,
+    })
+
+    const response = wrapResponse(this, tx, 'not applicable for declares')
```

Comment:
Yeah, I just kept it there cause it's weird to have nothing printed out on the command line. It might confuse users who are looking for an address and suspect something is wrong